### ISSUE TITLE
Multiple Share screen opens up 

### ISSUE THREAD

**Issue by Omegaviv** [CCExtractor/beacon#113]
If we click the share button multiple time at a time, multiple share screen opens up. As shown below in the demo video.

https://user-images.githubusercontent.com/86337946/152506231-3eb38d62-d2d8-45d3-8a13-6650f92d27c0.mp4

A simple fix to this can be that before opening check if already a share screen is open, close it and then open a new one.

is this the correct way, and should I work on this then??

**Comment by nb9960** [comment]
How would you check that?

**Comment by Omegaviv** [comment]
Yes that's what the problem is I am not finding anything that can check that

**Comment by Aadeesh11** [comment]
I think we can do something similar to [this](https://stackoverflow.com/questions/55634168/how-to-check-whether-alert-dialog-is-open-in-flutter), and hide the button when share screen is open and un-hide when the screen is closed.

**Comment by axl0d** [comment]
this happen when call from the `showBottomSheet` function, should call from scaffold `currentState.showBottomSheet`
